satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -1147,49 +1225,13 @@ datasets:
 
   qual:
     name: qual
+    long_name: Total Column Ozone Quality Index
+    standard_name: atmosphere_mass_content_of_ozone quality_flag
     key: '#1#totalOzone->totalOzoneQuality'
     resolution: 9001.209497451
-    standard_name: total_ozone_quality
     units: ""
     file_type: seviri_l2_bufr_toz
     coordinates:
        - longitude
        - latitude
     fill_value: 0
-
```

Comment:
do not remove these AMV datasets

Reply:
they also need a `long_name`

Reply:
These blocs from PR#2207 were added, plus long_name attribute